gradle/dependencies.gradle

```diff
@@ -187,9 +187,9 @@ libraries["jna"] = "net.java.dev.jna:jna:4.2.2"
 
 libraries["joda-time"] = "joda-time:joda-time:2.8.1"
 
-libraries["jsoup"] = "org.jsoup:jsoup:1.7.2"
+libraries["jsoup"] = "org.jsoup:jsoup:1.11.2"
 
-libraries["protobuf-java"] = "com.google.protobuf:protobuf-java:2.5.0"
+libraries["protobuf-java"] = "com.google.protobuf:protobuf-java:3.5.0"
```

Comment:
This one will be interesting. I vaguely remember John considering an upgrade to 3.X but shying away because it wasn't a drop-in replacement.